docs/DeveloperGuide.md

```diff
@@ -985,21 +985,27 @@ Priorities: High - (must have), Medium - (nice to have), Low -  (unlikely to hav
 1. Usability Requirements:
     * Should work on systems with Java 11 and above.
     * Should work on Windows, Linux and macOS.
+    * Should function fully offline without access to the internet.
+    * Cannot be used on mobile phones.
```

Comment:
Can change this to devices to make it applicable for other devices like ipad etc.
```suggestion
    * Cannot be used on mobile devices.
```